src/main/client/src/components/@pages/QRList.tsx

```diff
@@ -39,32 +41,52 @@ export const QRList: React.FC = (props) => {
     else return (100 * tokens.length) / totalTokenCount
   }
 
+  const alertBar = (acquired: number, needed: number) => {
+    if (acquired == null || needed == null) return <></>
+    else if (acquired < needed)
+      return (
+        <Alert variant="left-accent" status="info" mt="5">
+          <AlertIcon />
+          Még {needed - acquired} darab QR kód kell a tanköri jelenlét megszerzéséig.
+        </Alert>
+      )
+    else
+      return (
+        <Alert variant="left-accent" status="success" mt="5">
+          <AlertIcon />
+          Megvan a tanköri jelenlét!
+        </Alert>
+      )
+  }
+
   if (loading) return <Loading />
 
   return (
     <Page {...props} loginRequired>
-      <Heading>QR kód vadászat</Heading>
+      <Heading as="h1">QR kód vadászat</Heading>
+      {alertBar(tokens.length, minTokenToComplete)}
       <Paragraph>
-        A standoknál végzett aktív tevékenyégért QR kódokat lehet gyűjteni, amiket tanköri hiányzások igazolására lehet beváltani. TODO:
-        Fogalmazzunk meg ide szöveget, mert ma bevettem a fogalmazás gátlóm.
+        A standoknál végzett aktív tevékenyégért QR kódokat lehet gyűjteni. Ha eleget összegíűjtesz, beválthatod egy tanköri jelenlétre.
       </Paragraph>
 
-      <ButtonGroup>
+      <ButtonGroup mt="5">
         <LinkButton colorScheme="brand" leftIcon={<FaQrcode />} href="/qr/scan">
           QR kód beolvasása
         </LinkButton>
       </ButtonGroup>
 
-      <Heading>Haladás</Heading>
-      <Heading as="h4" size="md">
-        Eddig beolvasott tokenek: {tokens.length} / {totalTokenCount}
+      <Heading as="h3" mt="10">
```

Comment:
Add size explicitly. There is no global size definitions for headings for some reasons. Add a lg or smth. Now it maatches size with h1.